src/PostProcesses/RenderPipeline/Pipelines/ssao2RenderingPipeline.ts

```diff
@@ -292,7 +306,11 @@ export class SSAO2RenderingPipeline extends PostProcessRenderPipeline {
             effect.setFloat("near", this._scene.activeCamera.minZ);
             effect.setFloat("far", this._scene.activeCamera.maxZ);
             effect.setFloat("radius", this.radius);
-            effect.setTexture("depthSampler", this._depthTexture);
+            if (this._forceGeometryBuffer) {
+                effect.setTexture("depthNormalSampler", this._scene.enableGeometryBufferRenderer()!.getGBuffer().textures[0]);
+            } else {
+                effect.setTexture("depthNormalSampler", this._prePassRenderer.prePassRT.textures[2]);

```

Comment:
I wonder if we could add constants for the indices in the MRT as part of BABYLON.Constants ??? those gets inlined during build ;-)